types/react-native-push-notification/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-native-push-notification 7.3
+// Type definitions for react-native-push-notification 8.1
```

Comment:
Just noticed this line; you're bumping this up to another major version. Was that intentional?

If so, I believe there's a little more you have to do in order to keep v7 in the repo.

Reply:
> If so, I believe there's a little more you have to do in order to keep v7 in the repo.

This turned out to be wrong; they only have to be copied once someone wants to change v7 again.

In any case, let me know if this was intentional.